### ISSUE TITLE
Consider adding a MODULE_VERSION()?

### ISSUE THREAD

**Issue by edward-p** [llccd/netfilter-full-cone-nat#9]
If no version is set, dkms will have problem removing modules.
Although dkms has fixed this issue (https://github.com/dell/dkms/pull/291), but I still recommend adding a `MODULE_VERSION`

**Comment by kodatarule** [comment]
That is correct, I ran into issues installing new-lg4ff-dkms-git(from the aur repository) it kept giving me errors while downgrading to a previous dkms it allowed me to install it correctly.

Edit: **NVM apologies I didn't notice it is not for the DKMS git, opened an issue at the correct place.**
